content/javascript/concepts/arrays/terms/reduce/reduce.md

```diff
@@ -50,13 +50,22 @@ console.log(
   'Adding with initial value:',
   arrayTwo.reduce((acc, curr) => acc + curr, 'a')
 );
+
+// Add the values of each object inside an array.
+const arrayThree = [{x:1}, {x:2}, {x:4}];
+
+console.log(
+  'Adding object values:',
+  arrayThree.reduce((arr, curr) => acc + curr.x, 0)
```

Comment:
should this be "(acc, curr)" ?

Reply:
@SSwiniarski No problem thank you for the clarification - that should be it changed now.